sdk/python/feast/infra/offline_stores/contrib/spark_offline_store/spark_source.py

```diff
@@ -172,7 +175,9 @@ def get_table_query_string(self) -> str:
         """Returns a string that can directly be used to reference this table in SQL"""
         if self.table:
             # Backticks make sure that spark sql knows this a table reference.
-            return f"`{self.table}`"
+            # return f"`{self.table}`"
+            table = '.'.join(map(lambda x: f'`{x}`', self.table.split('.')))
```

Comment:
nit, `map` and `lambda` are generally avoided in favor of list comprehensions:
``
```suggestion
            table = '.'.join([f"`{x}`" for x in self.table.split('.')])
```